### PR TITLE
Chat

### LINKED ISSUE #103: Chat?

### PR DESCRIPTION
**Pull request by lionleaf**
Shrinequest.com has currently checked out this branch, so head over to check it out.

Fixes #103

### CODE REVIEW

**Comment by sigvef** [comment]
Can you rebase your commits to avoid unnecessary intermediate changes? ("it's no longer a hackathon :D")

**Comment by lionleaf** [comment]
ping.

Couldn't find a logical clean division, so it's now one commit.

File: client/src/Chat.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  body.appendChild(this.inputForm);
+  body.appendChild(this.chatWindow);
+}
+
+Chat.prototype.displayMessage = function(msg){
+  var p = document.createElement('p');
+  p.className = 'chat-msg';
+  p.textContent = msg;
+  this.chatWindow.appendChild(p);
+  console.log(p);
```
Comment (sigvef) [inline client/src/Chat.js]: Consider removing these console.logs.

**Comment by lionleaf** [comment]
bump. Added a console.log to the server instead; nice to have in the logs

**Pull request merged**